src/docker-osx-dev

```diff
@@ -992,28 +992,32 @@ function instructions {
 #
 function load_paths_from_docker_compose {
   local readonly yaml_file_path="$1"
-  local in_volumes_block=false
   local paths=()
 
-  if [[ -f "$yaml_file_path" ]]; then
-    local line=""
-    while read line; do
-      if $in_volumes_block; then
-        if [[ "${line:0:2}" = "- " ]]; then
-          local readonly path=$(echo $line | sed -ne "s/- \([^:]*\):.*$/\1/p" | sed -e 's/^["'\'']//'  -e 's/["'\'']$//')
-          if [ ! -z "$path" ]; then
-            paths+=("$path")
-          fi
-        else
-          in_volumes_block=false
+  local yaml_file_param=""
+  if [ ! -z "${yaml_file_path}" ]; then
+    yaml_file_param="-f ${yaml_file_path}"
+  fi
+
+  local in_volumes_block=false
+  local line=""
+  while read line; do
+    if $in_volumes_block; then
+      if [[ "${line:0:2}" = "- " ]]; then
```

Comment:
That is not really a change I made. I just changed the existing indentation when removing an if. 

Shouldn't there be another PR for changing condition code styles?